src/app/index.js

```diff
@@ -1,69 +1,137 @@
-var slideIndex, slides, dots;
-window.onload = function() {
+/**============= Require all the scss and js files =================== */
+import '../../style.scss';
+import { home } from '../app/home/home';
+import { products } from '../app/products/products';
+import { login } from '../app/login/login';
+import { signup } from '../app/signup/signup';
 
-  slides = document.getElementsByClassName("mySlides");
-  dots = document.getElementsByClassName("dot");
+let slideIndex, slides, dots;
 
+/**============= Handling page change on navigation click keeping header/footer fixed =================== */
+const routes = {
+  'signup': signup,
+  'signin': login,
+  'home': home,
+  'products': products
+}
+
+$(document).ready(function() {
+  slides = document.getElementsByClassName('mySlides');
+  dots = document.getElementsByClassName('dot');
+
+  // appending content section with home view(by default)
   content.innerHTML = routes['home'];
-  this.slideIndex = 1;
-  this.showSlides(this.slideIndex);
+
+  slideIndex = 1;
+  showSlides(slideIndex);
+
+  onInitSlider();
+
+  /**================== Navigation buttons ============================ */
+
+  $('.nav-home').on('click', function() {
```

Comment:
Below code for Navigation buttons is not optimal, there is lot of duplication that could have been avoided.